CMR-Metadata-Script/cmrmetadata.py

```diff
@@ -0,0 +1,65 @@
+import csv
+from datetime import datetime
+from datetime import timezone
+from pip._vendor import requests
+import uuid
+from config import BaseConfig
```

Comment:
Fix import order. Refer back to https://docs.google.com/document/d/1b0YSCObQu3yvWeblHDDeIKzapxUkuVQVElGw_rxrC4Q/edit .

Reply:
> Fix import order. Refer back to https://docs.google.com/document/d/1b0YSCObQu3yvWeblHDDeIKzapxUkuVQVElGw_rxrC4Q/edit .

Watch out for: Recommended to add [isort rule](https://github.com/PyCQA/isort/) to the pre-commit hook as well.